documents/papers/chep2019/modern-stack.tex

```diff
@@ -292,8 +295,8 @@ \subsection{SuperNEMO}
 some customised versions of particular packages, such as Qt, while relying on
 the main Spack repository for the vast majority of packages.
 
-At present, containers are built from scratch though use of a binary cache for this
-and eventual CVMFS deployment is in progress.
+At present, containers are built from scratch though the use of a binary cache
+to share build artefacts. Eventual deployment to CVMFS deployment is in progress.
```

Comment:
"to share build artefacts is under development."

Reply:
So how about, "At present, containers are built from scratch; the use of a binary cache to share build artefacts is under development, as is deployment to CVMFS." ?

Reply:
Yep, that's fine!